app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.aptoide.android.aptoidegames.feature_apps.presentation
+
+import androidx.compose.runtime.Composable
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.aptoide.android.aptoidegames.feature_apps.data.BonusData
+import kotlinx.coroutines.launch
+
+class BonusViewModel : ViewModel() {
+
+  fun addBundleData(
+    title: String,
+    tag: String
+  ) {
+    viewModelScope.launch {
+      BonusData.setBonusData(title, tag)
+    }
+  }
+
+  @Composable
+  fun rememberBonusData(): Pair<String, String> {
```

Comment:
This is a wrong way to inject and use view model.
Refer the other view models and remember functions syntax and usage.